component/publisher-client/src/test/java/org/wso2/am/analytics/publisher/DefaultResponseMetricBuilderTestCase.java

```diff
@@ -113,7 +117,7 @@ public void testMetricBuilder() throws MetricCreationException, MetricReportingE
                 .build();
 
         Assert.assertFalse(eventMap.isEmpty());
-        Assert.assertEquals(eventMap.size(), 26, "Some attributes are missing from the resulting event map");
+        Assert.assertEquals(eventMap.size(), 28, "Some attributes are missing from the resulting event map");
```

Comment:
Please assert the two values we are adding

Reply:
Fixed